### ISSUE TITLE
How did this recipe work for you? Ideas for improvement. 

### ISSUE THREAD

**Issue by stpaultim** [backdrop-contrib/faq_recipe#1]
If you used this recipe, please provide some feedback here. We'd love your suggestions on how to make this recipes more useful for other users.

**Comment by izmeez** [comment]
I haven't tried this recipe yet, and think i'd like to add the faq_ask module or make that a companion recipe.

**Comment by stpaultim** [comment]
> I haven't tried this recipe yet, and think i'd like to add the faq_ask module or make that a companion recipe.

If you are able to describe the difference and/or functionality involved here, that would be very helpful.

**Comment by izmeez** [comment]
The faq_ask is a separate drupal module that stalled in the migration from d6 to d7. It works with the faq module and brings a dynamic question answer workflow to build the faq. https://www.drupal.org/project/faq_ask

**Comment by Egmund** [comment]
I added a custom block at the bottom of 'main content' on faq page, but it could be an integrated part:
```
<p>
    <strong>Should you have any other questions, please do not hesitate to </strong><a href="/contact-us"><strong>contact us</strong></a><strong>.</strong>
</p>
```

**Comment by stpaultim** [comment]
@Egmund This could be added to the view as a footer to the view. 

I could put a generic block like this at the end of the FAQ. But, I can't link directly to a contact page, because I don't know what the site in question might be using for a contact page.

**Comment by Egmund** [comment]
That is true. I linked it to the 'Contact us webform recipe' installed with 'Digital Agency'.
It could be added with a note to re-set it to the contact page used by the site.
